### PR TITLE
Add --disable-mmap flag to RN50 data pipeline test

### PR DESCRIPTION
**Pull request by joehandzik**
Signed-Off-By: Joe Handzik <jhandzik@nvidia.com>

#### Why we need this PR?
- It adds the ability to disable mmap use within DALI readers for the RN50 pipeline test

#### What happened in this PR?
*Fill relevant points, put NA otherwise. Replace anything inside []*
 - What solution was applied:
     *[ New flag ]*
 - Affected modules and functionalities:
     *[ Just affects this file ]*
 - Key points relevant for the review:
     *[ N/A - very straightforward PR ]*
 - Validation and testing:
     *[ Tested against solutions at NVIDIA internal lab ]*
 - Documentation (including examples):
     *[ N/A. ]*

**JIRA TASK**: *[Use DALI-XXXX or NA]*

### CODE REVIEW

**Comment by JanuszL** [comment]
!build

File: dali/test/python/test_RN50_data_pipeline.py [hunk 1]
```diff
@@ -24,8 +24,8 @@
 
 class CommonPipeline(Pipeline):
     def __init__(self, data_paths, num_gpus, batch_size, num_threads, device_id, prefetch, fp16, random_shuffle, nhwc,
-                 decoder_type, decoder_cache_params, reader_queue_depth):
-        super(CommonPipeline, self).__init__(batch_size, num_threads, device_id, random_shuffle, prefetch_queue_depth=prefetch)
+                 dont_use_mmap, decoder_type, decoder_cache_params, reader_queue_depth):
+        super(CommonPipeline, self).__init__(batch_size, num_threads, device_id, random_shuffle, dont_use_mmap, prefetch_queue_depth=prefetch)
```
Comment (JanuszL) [inline dali/test/python/test_RN50_data_pipeline.py:28]:
```suggestion
        super(CommonPipeline, self).__init__(batch_size, num_threads, device_id, random_shuffle, prefetch_queue_depth=prefetch)
```
That breaks things

**Comment by joehandzik** [comment]
Overdid it when I migrated from the container hack - thank!

**Comment by JanuszL** [comment]
!build

**Pull request merged**